scripts/upload-compose-secrets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -eu
+
+if [ "${AWS_ACCOUNT}" = "dev" ]
+   then
+        export PASSWORD_STORE_DIR=${COMPOSE_PASSWORD_STORE_DIR}
+        COMPOSE_ACCESS_TOKEN=$(pass "compose/integration_tests/access_token")
+   else
+        export COMPOSE_PASSWORD_STORE_DIR?="${HOME}"/.paas-pass-high
+        COMPOSE_ACCESS_TOKEN=$(pass "compose/${AWS_ACCOUNT}/access_token")
+fi
+
+
+SECRETS=$(mktemp secrets.yml.XXXXXX)
+trap 'rm  "${SECRETS}"' EXIT
+
+cat > "${SECRETS}" << EOF
+---
+compose_access_token: ${COMPOSE_ACCESS_TOKEN}
```

Comment:
I'm inclined to stick with it as it is the pattern we have for other secrets in the repo and will prove useful if we need to pass in additional secrets.